src/components/core/geoChart/geoChartDataLayers.ts

```diff
@@ -46,17 +46,21 @@ function createPushpinSizeOptions(geoData: IGeoData): mapboxgl.Expression | numb
     // The mouseenter event uses queryRenderedFeatures to determine when the mouse is touching a feature
     // And queryRenderedFeatures is not supporting nested object in expression
     // https://github.com/mapbox/mapbox-gl-js/issues/7194
-    const sizeOptions: mapboxgl.Expression = ["step", ["get", "pushpinRadius"], DEFAULT_PUSHPIN_SIZE_VALUE];
+
+    const stepSizeOptions: mapboxgl.Expression = [
```

Comment:
Please use the name `sizeOptions` instead :)

Reply:
fixed